main.py

```diff
@@ -33,11 +33,11 @@ def __init__(self, conf: dict, hubitat):
         self.users = {}
         self.rejected_message = conf["rejected_message"]
         for group_name, group_data in conf["user_groups"].items():
-            is_admin = group_data["is_admin"]
+            is_admin = bool(group_data["is_admin"])
```

Comment:
https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
doesn't work when input is already a bool:
```
    self.case_insensitive = util.strtobool(conf["case_insensitive"])
[...]

util.py", line 314, in strtobool
    val = val.lower()
AttributeError: 'bool' object has no attribute 'lower'
```